dask/diagnostics/__init__.py

```diff
@@ -1,5 +1,6 @@
 from __future__ import absolute_import, division, print_function
 
+from ..callbacks import Callback
```

Comment:
Why this change?

Reply:
It seems sensible to me to have `Callback` available in the `dask.diagnostics` namespace.  Happy to revert though.

Reply:
Fine by me.